microprofile.jdt/com.redhat.microprofile.jdt.core/plugin.xml

```diff
@@ -19,6 +20,7 @@
       <delegateCommandHandler class="com.redhat.microprofile.jdt.internal.core.ls.MicroProfileDelegateCommandHandlerForJava">
             <command id="microprofile/java/codeLens"/>
             <command id="microprofile/java/hover"/>
+            <command id="projectIdentifier/identify"/>
```

Comment:
To be consistent with other commands, I will rename it to microprofile/java/projectNature

Reply:
for now, I changed the command name to `microprofile/java/projectNatures` in my latest push